### PR TITLE
Improve problem examples

### PR DESCRIPTION
**Pull request by pvdbosch**
solves #7

### CODE REVIEW

**Comment by pvdbosch** [comment]
OpenAPI 2.0 changes still to do, after 3.0 changes have been reviewed

**Comment by pvdbosch** [comment]
@jpraet , could you do a final review before merging?

File: src/main/openapi/problem/v1/problem-v1.yaml [hunk 1]
```diff
@@ -56,13 +56,13 @@ components:
           description: An absolute URI that identifies the specific occurrence of the problem. It may or may not yield further information if dereferenced.
       example:
         {
```
Comment (jpraet) [inline src/main/openapi/problem/v1/problem-v1.yaml]: Any particular reason why the Problem example is in JSON, while the InputValidationProblem example is in YAML?

File: src/main/openapi/problem/v1/problem-v1.yaml [hunk 2]
```diff
@@ -56,13 +56,13 @@ components:
...
-          "type": "urn:problem-type:belgif:payloadTooLarge",
-          "href": "https://www.belgif.be/specification/rest/api-guide/problems/payloadTooLarge.html", # location of linked doc will change in the future to recommended URI structure
-          "title": "Payload Too Large",
-          "status": 413,
-          "detail": "Request message must not be larger than 10 MB",
-          "instance": "urn:uuid:123e4567-e89b-12d3-a456-426614174000",
-          "limit": 10485760  # additional properties specific to the problem type are allowed
+          "type": "urn:problem-type:exampleOrganization:exampleProblem", # "exampleOrganization" should be a short identifier for the organization that defines the problem type. "belgif" is used for problem types standardized in the Belgif REST guide
```
Comment (jpraet) [inline src/main/openapi/problem/v1/problem-v1.yaml]: I remember a discussion about duplication of these comments of the example with the description of the properties. I think the idea was to align them?

Currently the example comments contain additional useful information, which is only visible when reading the source code. The description of the properties is more visible in tools like swagger-ui.

File: src/main/openapi/problem/v1/problem-v1.yaml (hunk 1 above)
Comment (pvdbosch) [reply src/main/openapi/problem/v1/problem-v1.yaml]: historical; as most examples seem to be YAML, I converted it to YAML as well.
SwaggerUI will convert it when displaying.

File: src/main/openapi/problem/v1/problem-v1.yaml (hunk 2 above)
Comment (pvdbosch) [reply src/main/openapi/problem/v1/problem-v1.yaml]: It was only said not to duplicate the same documentation twice.
I pushed a new commit though, to put more in the descriptions.

The remaining documentation in code comments is meant for API providers, so it should be OK if it's hidden in SwaggerUI, e.g. structure of the problem type URN shouldn't be interpreted (parsed) by clients, and the explanation "location within the body" only applies to the "name" property in the example; the JSON path structure isn't a guideline.

**Review by jpraet — APPROVED** [review]

**Comment by pvdbosch** [comment]
thanks for the review @jpraet , I'll wait for publication until next WG though, bc changes have to be aligned with possible feedback of the https://github.com/belgif/rest-guide/pull/155 review

**Comment by pvdbosch** [comment]
reviewed by WG, OK to merge

**Pull request merged**